Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -34,15 +34,18 @@ final class PackageToolTests: XCTestCase {
     }
 
     func testUsage() throws {
-        XCTAssert(try execute(["--help"]).stdout.contains("USAGE: swift package"))
+        let stdout = try execute(["-help"]).stdout
```

Comment:
The use of a single dash here is intentional so that we test both `--help` and `-help`.  The build tool test already did this, the others did not.